docs/Xamarin.Forms.Core/Xamarin.Forms.Internals/DependencyResolver+ResolveDelegate.xml

```diff
@@ -0,0 +1,31 @@
+<Type Name="DependencyResolver+ResolveDelegate" FullName="Xamarin.Forms.Internals.DependencyResolver+ResolveDelegate">
```

Comment:
We might consider if some dev time now drafting some isense docs might preempt questions/confusion in the forums.